composer/utils/analytics_helpers.py

```diff
@@ -0,0 +1,40 @@
+# Copyright 2024 MosaicML Composer authors
+# SPDX-License-Identifier: Apache-2.0
+
+"""Helpers for logging analytics with the MosaicMLLogger."""
+
+from composer.loggers.cometml_logger import CometMLLogger
```

Comment:
this is very brittle, and also its better to not have the global utils rely on the library itself. Can we just log the class names no matter what instead of this hardcoded list?